package.json

```diff
@@ -15,25 +15,31 @@
     "test": "mocha 'spec/javascripts/**/**spec.js?(x)' 'app/javascript/packages/**/*.spec.js?(x)'",
     "es5-safe": "is-es5-safe 'public/packs?(-test)/js/*.js'",
     "normalize-yaml": "normalize-yaml",
-    "build": "true"
+    "clean": "rm -f public/packs/*",
```

Comment:
This comment applies to an earlier version of the branch, and the command was changed in 2b9c52aea741aa311a886a1c138799709c6708ea . Is it still an issue in the latest version?

I had also been contemplating to change to just `rm -rf public/packs`, since the `*` was relevant mostly when the files were output into a flattened folder.

Reply:
Ah, yep, not an issue now. Dropping the `*` means it'll delete the `packs` folder, but I'm guessing that's fine?

Reply:
> Dropping the `*` means it'll delete the `packs` folder, but I'm guessing that's fine?

Good point. I think Webpack would be fine with that, as it should create the directory automatically during build. Off the top of my head, I can't think of anything else that might assume that directory to necessarily exist.

Reply:
I think I'll keep this as-is, since it's less prone to potential conflicting assumptions.